### DISCUSSION TITLE
I would like to ask a question about the invisible button in Karabiner-EventViewer.

### DISCUSSION THREAD

**Discussion by kyoox97** [pqrs-org/Karabiner-Elements#3616] [Q&A]
![스크린샷 2023-09-22 12 07 copy](https://github.com/pqrs-org/Karabiner-Elements/assets/147300322/31d68cc8-d6fd-4f78-8fe2-18fcc8fcb950)

Hello, I am a Korean illustrator.
I am enjoying using your carabiner app. thank you

I'm writing here because I have a question. Because I'm not a programmer
I don't know if I can write well, but I really hope you can answer.

I purchased a joystick called 8bitdo lite for drawing, and I wanted to map the keys on the joystick and use them as shortcuts in Photoshop.
I have successfully connected my Mac (OS Ventura) via Bluetooth.

As you know, we can check the button being pressed by checking the event viewer on the karabiner element. However, among the buttons on the connected 8bitdo lite, there are some that can be pressed and some that do not respond when pressed. You can check it by looking at the attached file.

What I want is to map all the buttons on 8bitdo lite.

If anyone knows a solution, I would greatly appreciate an answer.
I hope everyone has a good day today.

**Comment by 3170** [comment]
Hi there, the 8BitDo can be run in different modes. As from my knowledge you have to set it up to run in keyboard mode. Then all keys should be available.
